client/jetpack-cloud/sections/partner-portal/issue-license-v2/lib/incompatible-products.ts

```diff
@@ -0,0 +1,88 @@
+import { APIProductFamilyProduct } from 'calypso/state/partner-portal/types';
+import getProductInfo from '../../lib/get-product-info';
+
+/* Helper function to get the product slug. Some of lower tier products is replaced with higher tier products.
```

Comment:
I'd suggest making this comment more clear to express the intentions of this function.

Reply:
Totally agree, I added more context and examples in the description to explain the purpose of the function. 👍